09-schema/03-rules.md

```diff
@@ -80,56 +117,80 @@ DefineQuery query = Graql.define(
 **For those developing with Client [Python](/docs/client-api/python)**: Executing a `define` query, is as simple as passing the Graql(string) query to the [`query()`](/docs/client-api/python#client-api-method-lazily-execute-a-graql-query) method available on the [`transaction`](/docs/client-api/python#client-api-title-transaction) object.
 </div>
 
-The Graql rule above is telling Grakn the following:
+Please note that facts defined via rules are in general not stored in the knowledge graph. In this example, siblings data is not explicitly stored anywhere in the knowledge graph. However by defining the rule in the schema, at query time the extra fact will be generated so that we can always know who the siblings are.
 
-when
-: there are two different people (`$x` and `$y`) who have the same mother `$m` and the same father `$f`,
+<!-- This is a basic example of how Graql rules can be useful. In a dedicated section, we learn about rules by looking at more examples of [rule-based automated reasoning](...). -->
 
-then
-: those people (`$x` and `$y`) must be siblings.
+## Retrieving rules
 
-If you find the Graql code above unfamiliar, don't be concerned. We soon learn about [using Graql to describe patterns](/docs/query/match-clause).
+To retrieve rules, we refer to them by their label in a match statement:
 
-In this example, siblings data is not explicitly stored anywhere in the knowledge graph. But by having included this rule in the schema, we can always know who the siblings are and use the `siblings` relationship in our queries.
+```
+match $x label people-with-same-parents-are-siblings; get;
+```
 
-<!-- This is a basic example of how Graql rules can be useful. In a dedicated section, we learn about rules by looking at more examples of [rule-based automated reasoning](...). -->
+## Deleting rules
 
-## The Underlying Logic
-Under the hood, rules are restricted to be definite Horn Clauses. In simple terms, the Graql statements placed in the `when` body form one single condition where all statements must be true for the rule to apply. The `then` body, on the other hand, is restricted to contain one single statement only.
+To delete rules we refer to them by their label and use the undefine keyword. For the case of the rules defined above, to delete them we write:
 
-To simplify this logic even further, you can think of the [siblings example](#define-a-rule) in form of an `if` statement like so:
+```
+undefine people-with-same-parents-are-siblings sub rule;
+```
+
+To persist the result, executing a commit; statement is also required.
```

Comment:
isn't `commit` a console command? if so, it goes without `;`.
also, we haven't mentioned `commit` when _defining_ a rule.
lastly, there is this _colored panel_ used elsewhere if you wanna reuse it here:
```
[Important] Don't forget to `commit` after executing a `define` query. Otherwise, anything you have defined is NOT committed to the original keyspace that is running on the Grakn server. When using one of the Grakn Clients, to commit changes, we call the `commit()` method on the `transaction` object that carried out the query. Via the Grakn Console, we use the `commit` command.
```
(It may come up with some sort of referencing markdown syntax to easily include snippets of text  like this one.)